pipelines/rj_ssm__celular_seguro/flow.py

```
@@ -0,0 +1,46 @@
+    response = requests.post(url,
+                            json=payload,
+                            verify=False,
+                            cert=cert,
+                            headers=headers
+                            )
+
+    print("Status:", response.status_code)
+    print("Resultado:", response.json())
+
+
+    requests.post()
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)